### PR TITLE
testing article model

### PR DESCRIPTION
**Pull request by BehnazDer**
New tests on article model

### CODE REVIEW

**Comment by BehnazDer** [comment]
There is no failing when I yarn test, i don't know why there is a failure in CI

File: src/middleware/verifyToken.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const verifyToken = (req, res, next) => {
     return;
   }
   try {
-    const verified = jwt.verify(token, 'SECRET_KEY');
+    const verified = jwt.verify(token,'JWT_SECRET');
```
Comment (thdvbr) [inline src/middleware/verifyToken.js]: this should be process.env.JWT_SECRET, not strings

Comment (thdvbr) [reply src/middleware/verifyToken.js]: also check/test after changing if everything is working fine

File: src/utils/authentication.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const generateToken = (user) => jwt.sign(
   {
     _id: user._id, email: user.email, username: user.username, role: user.role,
   },
-  'SECRET_KEY',
+  'JWT_SECRET',
```
Comment (thdvbr) [inline src/utils/authentication.js]: here as well

File: src/models/article.model.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ const articleSchema = new mongoose.Schema(
     violence_type: {
       type: [String],
       required: true,
+      validate: (v) => v == null || (v.length > 0),
```
Comment (irina-baeva) [inline src/models/article.model.js]: may be specify what is v here?

File: src/models/__mocks__/article.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+export const normalArticle = {
```
Comment (thdvbr) [inline src/models/__mocks__/article.js]: i added naming convention for mocks on our readme, we will now call mocks like mock*** .. for instance change this to mockArticle and change all the other stuff

File: src/models/article.model.js (hunk 3 above)
Comment (BehnazDer) [reply src/models/article.model.js]: what do u mean exactly? writing validator instead of v ?

Comment (irina-baeva) [reply src/models/article.model.js]: i just did not understand what is "v"? is it "validator"? i thought it is "value" or something

Comment (irina-baeva) [reply src/models/article.model.js]: but never mind, it is just a small comment cuz it is not clear. But lets change in this PR mock names (like @thdvbr commented) for your article test and merge, so we can see what's left.
Do not forget to check lint (CI is not working for this command now)

File: src/models/__mocks__/article.js (hunk 4 above)
Comment (BehnazDer) [reply src/models/__mocks__/article.js]: changed

File: src/models/article.model.js (hunk 3 above)
Comment (BehnazDer) [reply src/models/article.model.js]: changed

**Pull request merged**